src/elements.tsx

```diff
@@ -104,6 +104,8 @@ const isVoidElement = (tagName: string) => {
 };
 
 export function createElement(name: string, attributes: Attributes | undefined, ...contents: string[]) {
```

Comment:
The type annotation of the `name` parameter is now `string | { (attributes: Attributes | undefined, contents: string[]): string }`  right? Could you also update that?

Reply:
I had a question about the updated interface for createElements's name parameter. I was going to make a new type for it as well as export Attributes to reuse it for the custom elements. Should Attributes go in a new file?